autoblocks/_impl/tracer.py

```diff
@@ -112,27 +198,49 @@ async def _send_event_unsafe(
         timestamp: Optional[str] = None,
         properties: Optional[Dict] = None,
         prompt_tracking: Optional[Dict] = None,
+        evaluators: Optional[List[BaseEventEvaluator]] = None,
```

Comment:
we can actually drop the default args (all the Nones) for everything in this function since we pass everything explicitly when calling from `send_event`